### PR TITLE
Use UNUSED macro from RCLC

### PR DESCRIPTION
**Pull request by pablogs9**
Related: https://github.com/micro-ROS/freertos_apps/pull/36

### CODE REVIEW

**Comment by pablogs9** [comment]
Build failed because generated libraries do not include these changes: https://github.com/micro-ROS/rclc/pull/41.

We should generate a sort of Github action that generates and releases a version of this repo and ideally it should be listening to changes in a list of relevant repos (rclc, rmw...)

File: examples/micro-ros_tf_publisher/micro-ros_tf_publisher.ino [hunk 1]
```diff
@@ -27,7 +27,7 @@ rcl_timer_t timer;
 #define RCCHECK(fn) { rcl_ret_t temp_rc = fn; if((temp_rc != RCL_RET_OK)){error_loop();}}
 #define RCSOFTCHECK(fn) { rcl_ret_t temp_rc = fn; if((temp_rc != RCL_RET_OK)){error_loop();}}
 
-extern "C" int clock_gettime(clockid_t unused, struct timespec *tp);
+extern "C" int clock_gettime(clockid_t RCLC_UNUSED, struct timespec *tp);
```
Comment (jamoralp) [inline examples/micro-ros_tf_publisher/micro-ros_tf_publisher.ino:30]: Ctrl + H?

Comment (pablogs9) [reply examples/micro-ros_tf_publisher/micro-ros_tf_publisher.ino:30]: Sorry sorry

Comment (jamoralp) [reply examples/micro-ros_tf_publisher/micro-ros_tf_publisher.ino]: I would've done the same :slightly_smiling_face:

**Pull request merged**